inc/console/migration/timestampscommand.class.php

```diff
@@ -48,39 +48,39 @@ protected function configure() {
       parent::configure();
 
       $this->setName('glpi:migration:timestamps');
-      $this->setDescription(__('Convert DATETIME to timestamps to use timezones.'));
+      $this->setDescription(__('Convert "datetime" fields to "timestamp" to use timezones.'));
    }
 
    protected function execute(InputInterface $input, OutputInterface $output) {
       //convert db
 
       // we are going to update datetime, date and time (?) types to timestamp type
       $tbl_iterator = $this->db->request([
-         'SELECT'       => ['INFORMATION_SCHEMA.COLUMNS.TABLE_NAME'],
+         'SELECT'       => ['information_schema.COLUMNS.TABLE_NAME'],
          'DISTINCT'     => true,
-         'FROM'         => 'INFORMATION_SCHEMA.COLUMNS',
+         'FROM'         => 'information_schema.COLUMNS',
          'INNER JOIN'   => [
-            'INFORMATION_SCHEMA.TABLES' => [
+            'information_schema.TABLES' => [
                'ON' => [
-                  'INFORMATION_SCHEMA.TABLES.TABLE_NAME',
-                  'INFORMATION_SCHEMA.COLUMNS.TABLE_NAME', [
-                     'AND' => ['INFORMATION_SCHEMA.TABLES.TABLE_TYPE' => 'BASE TABLE']
+                  'information_schema.TABLES.TABLE_NAME',
+                  'information_schema.COLUMNS.TABLE_NAME', [
+                     'AND' => ['information_schema.TABLES.TABLE_TYPE' => 'BASE TABLE']
                   ]
                ]
             ]
          ],
          'WHERE'       => [
-            'INFORMATION_SCHEMA.COLUMNS.TABLE_SCHEMA' => $this->db->dbdefault,
-            'INFORMATION_SCHEMA.COLUMNS.COLUMN_TYPE'  => 'DATETIME'
+            'information_schema.COLUMNS.TABLE_SCHEMA' => $this->db->dbdefault,
+            'information_schema.COLUMNS.DATA_TYPE'    => 'datetime'
```

Comment:
I used the `DATA_TYPE` here as `The DATA_TYPE value is the type name only with no other information. The COLUMN_TYPE value contains the type name and possibly other information such as the precision or length.`

See https://dev.mysql.com/doc/refman/8.0/en/columns-table.html

Reply:
MySQL does not care about case; postgresql do; and all tables/columns names are lowercase. I prefer to keep all lowercase to prevent confusions.

Reply:
OK, I will change it